hubspot3/tickets.py

```diff
@@ -78,3 +90,171 @@ def get_all(self, limit: int = -1, **options) -> list:
             offset = batch["offset"]
 
         return output if not limited else output[:limit]
+
+    default_batch_properties = [
+        "subject",
+    ]
+
+    def _get_batch(self, ids, extra_properties: Union[list, str] = None,
+                   with_history: bool = False):
+        """given a batch of vids, get more of their info"""
+        # default properties to fetch
+        properties = set(self.default_batch_properties)
+
+        # append extras if they exist
+        if extra_properties:
+            if isinstance(extra_properties, list):
+                properties.update(extra_properties)
+            if isinstance(extra_properties, str):
+                properties.add(extra_properties)
+        if with_history:
+            property_name = 'propertiesWithHistory'
+        else:
+            property_name = 'properties'
+        if len(properties) > 0:
+            params = {property_name: list(properties)}
+        else:
+            params = {}
+        batch = self._call(
```

Comment:
You might do this request using one line of code, I guess.

Reply:
Just following the original code